impeller/renderer/backend/gles/render_pass_gles.cc

```diff
@@ -207,13 +207,6 @@ void RenderPassGLES::ResetGLState(const ProcTableGLES& gl) {
   }
 
   GLuint fbo = GL_NONE;
-  fml::ScopedCleanupClosure delete_fbo([&gl, &fbo]() {
-    if (fbo != GL_NONE) {
-      gl.BindFramebuffer(GL_FRAMEBUFFER, GL_NONE);
```

Comment:
from looking at profiles binding this framebuffer actually triggers a driver workload to flush from fb0.